argo-event/templates/eventbus.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventBus
+metadata:
+  name: {{ .Values.name }}
+  {{- with .Values.labels }}
+  labels:
+    {{ toYaml . }}
+  {{- end }}
+spec:
+  jetstream:
+    version: {{ .Values.eventbus.config.version | default "latest" }}
+    {{- with .Values.eventbus.config.affinity }}
+    affinity:
+      {{ toYaml . }}
```

Comment:
reference to https://github.com/shoplineapp/helm-charts/blob/master/simple/templates/deployment.yaml#L67C6-L76C24